### DISCUSSION TITLE
OSPO Advisory Board at GHU 2023

### DISCUSSION THREAD

**Discussion by ahpook** [community/OSPO#96] [General]
If you're thinking about attending GitHub Universe, maybe this will tip the scales! We're hosting the second OSPO Advisory Board as a Day 0 event before Universe and would love to see you there. The format will be a mix of product/roadmap presentations from Hubbers and a peer-to-peer group discussion section in [open spaces format](https://devopsdays.org/open-space-format/) – which I fell in love with from devops days. 

Please drop me an email or comment here if you're interested. Note you'll still have to register separately for GHU itself, and make your travel plans so you'll arrive early enough to attend.

Feel free to use this discussion area to ask questions, plan ahead, and link up with fellow attendees!

**Comment by rrrutledge** [comment]
I am interested in being involved from an InnerSource perspective but will not be on-site.  Is this one open for virtual attendance?

**Reply by ahpook** [reply]
Hi @rrrutledge - yes, the presentation/Q&A portion will be remote-enabled. OK if I email you with the reg link and more details?

**Reply by rrrutledge** [reply]
Sounds good, @ahpook - thank you❗️

**Reply by ZulfaNasry** [reply]
it's great!

**Comment by ZulfaNasry** [comment]
Hello, I'm an undergraduate student, and I'm very interested in attending this event. May I ask if undergraduate students are allowed to participate in this event?

**Reply by ahpook** [reply]
@ZulfaNasry Thank you for the interest. I think you will get more value out of the GitHub Universe sessions. You can register for free here: https://reg.githubuniverse.com/flow/github/universe23/reg/login

**Reply by ZulfaNasry** [reply]
Thank you for the information. It appears that GitHub now requires company details, but as I'm currently a student without company affiliations, I may not have the necessary information to provide. I appreciate your assistance.

**Comment by cunla** [comment]
Hi, I would be happy to attend - please email me the details
